cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -174,7 +174,7 @@ private static File createLocalDir(CConfiguration cConf, String dirName) {
       File credentialsDir = new File(localDirStr, dirName);
 
       try {
-        java.nio.file.Files.createDirectories(credentialsDir.toPath(), FileUtils.OWNER_ONLY_RWX);
```

Comment:
The actual credential files written inside this directory are written with restrictive permissions, so the restrictive permissions on the directory aren't necessary in the same manner.